BoxContentSDK/BoxContentSDK/Models/BOXRecentItem.h

```diff
@@ -0,0 +1,41 @@
+//
+//  BOXRecentItem.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+
+/**
+ Represents an item recently interacted with by the current user.
+ */
+@interface BOXRecentItem : BOXModel
+
+/**
+ The file or bookmark that was recently interacted with by the user.
+ */
+@property (nonatomic, readwrite, strong) BOXItemMini *item;
```

Comment:
Seems like the recent_items endpoint respects the 'fields' param. Should we be inheriting from BOXItem instead?